src/blocks/map/components/inspector.js

```diff
@@ -197,7 +197,10 @@ class Inspector extends Component {
 							/>
 						</PanelBody>
 					}
-					<PanelBody title={ __( 'Google Maps API Key' ) } initialOpen={ false }>
+					<PanelBody
+						title={ __( 'Google Maps API Key' ) }
+						initialOpen={ address ? false : true }
```

Comment:
@richtabor I think we can simplify here `initialOpen={ !address }`

Reply:
Good catch 👌